vm_insnhelper.c

```diff
@@ -3711,6 +3711,14 @@ vm_call_method_nome(rb_execution_context_t *ec, rb_control_frame_t *cfp, struct
     }
 }
 
+static VALUE
+vm_defined_class_of_method_entry(const rb_callable_method_entry_t *me)
```

Comment:
The function name is a bit confusing. At least could you write a comment?

Reply:
👍 It is. I will think of a clearer name.

Reply:
Done